lib/watir-webdriver/checkers.rb

```diff
@@ -0,0 +1,88 @@
+module Watir
+  class Checkers
+
+    def initialize(browser)
+      @browser = browser
+      @error_checkers = []
```

Comment:
Since class name is already `Checkers` then having an instance variable called `@error_checkers` seems to be too much. Maybe `hooks` if you rename the checkers in the first place.